src/main/java/com/pm/backend/model/v1/group/GroupModel.java

```diff
@@ -0,0 +1,28 @@
+package com.pm.backend.model.v1.group;
+
+
+import com.pm.backend.model.v1.user.UserModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import lombok.experimental.Accessors;
+import org.springframework.data.mongodb.core.mapping.DBRef;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.HashMap;
+import java.util.List;
+
+
+@Data
+@Document(collection = "groups")
+@Accessors(chain=true)
+public class GroupModel {
+    //@ApiModelProperty(hidden = true)
```

Comment:
Not sure what this annotation does exactly, but we probs don't want commented out code